lib/features/home/presentation/screen/home_screen.dart

```diff
@@ -1,19 +1,20 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:mentorship_e1_g3/core/di/dependency_injection.dart';
 import 'package:mentorship_e1_g3/core/themes/app_pallete.dart';
-import 'package:mentorship_e1_g3/features/home/presentation/cubit/cubit/home_cubit.dart';
 import 'package:mentorship_e1_g3/features/home/presentation/widgets/bottom_nav_bar_items_list.dart';
+import '../../logic/cubit/home_cubit.dart';
 
 class HomeScreen extends StatelessWidget {
   const HomeScreen({super.key});
 
   @override
   Widget build(BuildContext context) {
+    final HomeCubit cubit=getIt.get<HomeCubit>();
     return BlocProvider(
-      create: (context) => HomeCubit(),
+      create: (context) => cubit,
```

Comment:
if u not gonna use this "cubit" instance later, u can make it like that :

create: (context) => getIt< HomeCubit >(),

instead of : 

create: (context) => cubit,